pkg/virt-launcher/virtwrap/converter/converter_test.go

```diff
@@ -917,13 +917,15 @@ var _ = Describe("Converter", func() {
 				Expect(domainSpec.VCPU.CPUs).To(Equal(uint32(3)), "Expect vcpus")
 			})
 
-			It("should define hotplugable default topology", func() {
+			DescribeTable("should define hotplugable default topology", func(arch string) {
 				v1.SetObjectDefaults_VirtualMachineInstance(vmi)
 				vmi.Spec.Domain.CPU = &v1.CPU{
 					Cores:      2,
 					MaxSockets: 3,
 					Sockets:    2,
 				}
+				c.Architecture = NewArchConverter(arch)
+				vmiArchMutate(arch, vmi, c)
```

Comment:
I think this is not needed and can be dropped

Reply:
Updated